site/docs/components/tooltip/accessibility.mdx

```diff
@@ -8,6 +8,56 @@ data:
   $ref: ./#/data
 ---
 
+Digital accessibility is a shared responsibility and while the standard behavior and attributes for a tooltip are already handled for you as part of our component, product teams are responsible for implementing the component correctly and ensuring their digital products meet all current accessibility requirements.
+
+## What is a tooltip
+A tooltip is a non-modal overlay that is hidden by default and displays text-only content on hover or focus. It should only be used to provide extra (non-essential) information about a [`Form Field`](../form-field), [`Link`](../link), [`Button`](../button), or other interactive element.
+
 ## Best practices
+- Tooltips are triggered by both focus and hover events and remain visible and persistent until:
+  - Hover or focus is removed from the trigger element
+  - The user manually dismisses it (e.g.`Esc` key)
+  - The content is no longer valid 
+- Focus does not move to the tooltip. Since a tooltip does not receive focus, do not include interactive elements in the tooltip as the user cannot access them.
+- Only one tooltip should be displayed at a time.
+- Keep the text for a tooltip short and concise.
+- Avoid rich content such as bold text, italics, headings, icons, etc. as these will not be conveyed through `aria-describedby`.
+- Do not put essential information in tooltips.
+- Do not use a timeout to hide the tooltip.
+- The expand/collapsed state is intentionally not exposed.
+- Keep the use of tooltips to a minimum and use only when providing extra (non-essential) information for the user.
```

Comment:
This is similar to an above point:

"Do not put essential information in tooltips."